OSDOCS-2478: Adds procedure for enabling the multicluster console (#42063)
openshift/openshift-docs: modules/enabling-multi-cluster-console.adoc

// Module included in the following assemblies:
//
// * assemblies/web-console.adoc

:_content-type: PROCEDURE
[id="enable-multi-cluster-console_{context}"]

> = Enabling multi-cluster in the web console

Reviewer: Should add in a description of what multi-cluster does and why you might want to enable it?

:FeatureName: Multi-cluster console
include::snippets/technology-preview.adoc[leveloffset=+1]
//

.Prerequisites
* You must have link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html/install/index[Red Hat Advanced Cluster Management(ACM) for Kubernetes 2.5] installed.

* You must have upgraded to the latest version of {product-title}.

.Procedure

. Log in to the {product-title} web console using your login credentials.

> . Enable the feature gate from *Cluster Settings* -> *Configuration* -> *FeatureGate*, and click on *enable*.

Reviewer: @opayne1 I would add a warning here saying that this should not be set on production clusters. You will not be able to upgrade your cluster after applying the feature gate, and it can be undone.

Reviewer: The steps are a little different. Currently you have to edit the YAML. It's currently documented here:

https://docs.openshift.com/container-platform/4.9/nodes/clusters/nodes-cluster-enabling-features.html

:_content-type: PROCEDURE
[id="enable-multi-cluster-console_{context}"]
= Enabling multicluster in the web console

:FeatureName: Multicluster console
include::snippets/technology-preview.adoc[leveloffset=+1]
//

.Prerequisites
* You must have link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html/install/index[Red Hat Advanced Cluster Management (ACM) for Kubernetes 2.5] installed.
* You must have upgraded to the latest version of {product-title}.
* You must have administrator privileges.

WARNING: This should not be set on production clusters. You will not be able to upgrade your cluster after applying the feature gate, and it cannot be undone.

.Procedure

. Log in to the {product-title} web console using your credentials.

. Enable ACM in the administrator perspective by following *Admin* -> *ClusterSettings* -> *Configuration* -> *Console Details* -> *Console Plugins* and click `enable`.

. Click *Refresh the web console* in the pop-up window that states that a web console update is available.

. *local-cluster* and *All Clusters* is visible above the perspectives in the navigation section.

> . Enable the feature gate from *Administration* -> *Cluster Settings* -> *Configuration* -> *FeatureGate*, and edit the YAML.

Reviewer 2: ```suggestion
. Enable the feature gate from *Administration* -> *Cluster Settings* -> *Configuration* -> *FeatureGate*, and edit the YAML template as follows:
```
Does editing the YAML enable the feature gate or is there a button to click first? I am unsure if it should be "... *FeatureGate* by editing the YAML <template/markup/file> as follows:"

+
[source,yaml]

----
spec:
    featureSet: TechPreviewNoUpgrade
----

> . Click save. This enables for all clusters.

Reviewer 2: ```suggestion
. Click save to enable the Multicluster console for all clusters.
```
See the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references)'s guidance on pronouns. I am not sold on my suggestion, but you can probably do better.

> IMPORTANT: Once you save, this cannot be undone.

Reviewer 2: ```suggestion
[IMPORTANT] 
====
After you save, this feature is enabled and cannot be undone.
====
```
Re: [ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#o) guidance on _once_ and pronoun usage.
